Jenkinsfile

```diff
@@ -184,28 +184,6 @@ pipeline {
                         }
                     }
                 }
-                stage("address sanitizer") {
-                    steps {
-                        container('autopas-gcc7-cmake-make') {
-                            dir("build-addresssanitizer") {
-                                sh "cmake -DCCACHE=ON -DCMAKE_BUILD_TYPE=Debug -DAUTOPAS_ENABLE_ADDRESS_SANITIZER=ON .."
-                                sh "entrypoint.sh make -j 4 > buildlog.txt 2>&1 || (cat buildlog.txt && exit 1)"
-                                sh './tests/testAutopas/runTests'
-                            }
-                        }
-                    }
-                }
-                stage("address sanitizer release") {
-                    steps {
-                        container('autopas-gcc7-cmake-make') {
-                            dir("build-addresssanitizer-release") {
-                                sh "cmake -DCCACHE=ON -DCMAKE_BUILD_TYPE=Release -DAUTOPAS_ENABLE_ADDRESS_SANITIZER=ON .."
-                                sh "entrypoint.sh make -j 4 > buildlog.txt 2>&1 || (cat buildlog.txt && exit 1)"
-                                sh './tests/testAutopas/runTests'
-                            }
-                        }
-                    }
-                }
```

Comment:
you remove this because it seems redundant with the clang address sanitizer stages?

Reply:
yes